### DISCUSSION TITLE
strongswan on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by GKJoker98** [strongswan/strongswan#1036] [Q&A]
Anyone else having problems with this setup (ubuntu 22.04 Server on raspberry pi 3+)? If I use "ipsec up ikev2-rw" and "ipsec statusall" nothing happens (blank line). I recognized that I cant start strongswan-starter. I get the error "cant load critical plugin".

I used the guide on [digitalocean ](https://www.digitalocean.com/community/tutorials/how-to-set-up-an-ikev2-vpn-server-with-strongswan-on-ubuntu-22-04)

**Comment by tobiasbrunner** [comment]
These commands interact with the IKE daemon, which they can't if it isn't running. So fix that error (read the log).
